documentation/internal/Hardening.md

```diff
@@ -82,13 +83,16 @@ metadata:
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
-  name: view
+  name: healthz
 subjects:
 - kind: ServiceAccount
   name: healthz
   namespace: kube-system
 ```
 
+**Note:** ClusterRole and ClusterRoleBinding are not required
+if you have `system:discovery` or `system:public-info-viewer` ClusterRoleBindings installed on the cluster (default).
```

Comment:
The following rolebindings provide wider permissions as it's necessary for the current case. It should be reflected in the article, since it's a security hardening guide.

Reply:
Clarified in https://github.com/Netcracker/KubeMarine/pull/588/commits/27ec063f246f332cd1262724bdd9fad2d2eb7a36